ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml

```diff
@@ -77,14 +75,29 @@ tests:
 - as: e2e-metal-assisted-ipv6
   steps:
     cluster_profile: packet
-    dependencies:
-      ASSISTED_AGENT_IMAGE: pipeline:assisted-installer-agent
     env:
       ASSISTED_CONFIG: |
         IPv6=yes
         TEST_FUNC=test_install
         MAKEFILE_TARGET="create_full_environment run test_parallel"
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-single-node
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        NUM_MASTERS=1
+        TEST_FUNC=test_install
+        MAKEFILE_TARGET="create_full_environment run test_parallel"
+    workflow: baremetalds-assisted-install
+- as: system-test-olm
```

Comment:
why should the naming between the two jobs be so different?
I don't see any justification to call it ``system-test-olm``

Reply:
It was named like this to differentiate between the old workflows running `start_discovery.py` to the pytest workflow.
Since we changed it already, I agree it should be aligned like the other tests to have an `e2e` prefix instead of `system`.

I'll make this change on a separate PR to make it on all the repositories. Currently this PR only adds new tests.